src/pages/community/foundation/local-initiative.mdx

```diff
@@ -0,0 +1,136 @@
+framework for organizing local GraphQL user groups and offers support each approved
+group (hereafter, a "Local"). Here’s what you need to know to get involved:
+
+## Requirements
+
+Organizers of a Local must agree:
+
+- All official events and related channels will be governed by the
+  **[GraphQL Code of Conduct](https://graphql.org/codeofconduct/)**.
+- Official communications will comply with the
+  **[GraphQL Brand Guidelines](https://graphql.org/brand/)**.
+- The Local accepts **GraphQL TSC oversight**:
+  - At least one
+    [GraphQL Technical Steering Committee (TSC) member](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#tsc-members)
+    must have administrator access to the Local’s online platforms.
+  - In case of a serious Code of Conduct breach, an organizer may be replaced.
+- The Local has 1 or more organizer from the area. (We're pretty flexible on the
+  definition of "from the area", just be reasonable.)
+- Each official event will have at least least one **organizer present**.
+- Official events will be held in an appropriate **professional setting**.
+- Official events will feature at least one GraphQL-related talk (the format can
+  vary: lecture, lightning talk, panel, AMA, etc).
+- These **talks will be recorded** and made freely available online.
+- Official events will be be **announced at least two weeks in advance**.
+  Popular ticketing platforms include Eventbrite, Guild.host, and Meetup.com.
+- Attending official events will be **affordable**.
+  - If not free, attendance fees must not exceed USD $10 or local equivalent
+    without explicit approval from the GraphQL Marketing and Content committee.
+    Events charging fees waive their entitlement to GraphQL Foundation
+    sponsorship.
+- Organizers will **report back on attendance**, general feedback, and any
+  suggestions for future events in a timely manner.
+
+Groups may of course organize other events, e.g. a summer barbeque, but these
+are currently out-of-scope of GraphQL Foundation support. These other events
+shall not be referred to as "official", but are still encouraged to follow the
+guidelines that are applicable (in particular, the Code of Conduct and Brand
+Guidelines must still be honoured).
+
+## Support
+
+Each approved Local is entitled to request the following support for each
+event:
+
+- **Sponsorship** up to USD $200 per event (max 1 per quarter) for approved
+  costs (see below). More sponsorship might be available directly from
+  individual foundation members, see details below.
+- **Swag bundle**\* to distribute to speakers, attendees, or organizers (max 1
+  per quarter).
+- **Google Slides templates** for:
+  - Code of Conduct
+  - Call for Papers (CFP)
+  - Sponsors
+  - Schedule
+  - Speaker/talk introduction
+- **Promotion** through GraphQL’s Twitter and LinkedIn channels.
+- **Custom artwork** adhering to the GraphQL Brand Guidelines; including
```

Comment:
I think we want to be clear that this is not, "we will make custom graphics for each of your meetups" - I think we should just call this Local Artwork so that we don't indicate that we are going to do custom graphic design work

Reply:
Good point; have renamed to "Local artwork" throughout.